### DISCUSSION TITLE
Preview powerpoint/pdf files on page

### DISCUSSION THREAD

**Discussion by RDeveloper6** [requarks/wiki#3215] [Help / Questions] [answered]
Is there a way to upload files like powerpoint/pdf/word..
and have feature to see a preview of the files and not just link foe downloading it..
Like in Confluence made by Atlassian, that's allow to open the file (If supported) in a preview like window

**Comment by NGPixel** [comment, marked as answer]
Not at the moment, suggest a feature on https://feedback.js.wiki/wiki
